### PR TITLE
DGC-291: Create announcements view.

### PR DESCRIPTION
**Pull request by phernand42**

### CODE REVIEW

File: config/default/views.view.special_announcements_display.yml [hunk 1]
```diff
@@ -782,13 +687,518 @@ display:
...
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          empty: false
+          tokenize: false
+          content:
+            value: '<a href="special-announcements">More Announcements</a>'
```
Comment (lkopacz) [inline config/default/views.view.special_announcements_display.yml]: @phernand42 can we assure this is a relative link? `/special-announcements` ?

**Comment by lkopacz** [comment]
@phernand42, I left one small comment and this needs to be rebased again. I hate it when you merge one new thing and have to rebase again!

**Comment by phernand42** [comment]
@lkopacz PR rebased and changed applied. Let me know if there are any other changes that need to be made.

**Comment by phernand42** [comment]
@lkopacz PR rebased

**Pull request merged**